app/assets/stylesheets/_variables.scss

```diff
@@ -0,0 +1,15 @@
+//
+// Variables
+// --------------------------------------------------
+
+
+//== Colors
+//
+//## Gray colors for use across Coursemology.
+
+$gray-base:              #000 !default;
```

Comment:
Since we are using "color" instead of "colour", so american ?  And bootstrap is using "gray" instead of "grey"